frontend/src/lib/constants.tsx

```diff
@@ -185,7 +184,6 @@ export const FEATURE_FLAGS = {
     ERROR_TRACKING_ALERT_ROUTING: 'error-tracking-alert-routing', // owner: #team-error-tracking
     EXPERIMENT_INTERVAL_TIMESERIES: 'experiments-interval-timeseries', // owner: @jurajmajerik #team-experiments
     FEATURE_FLAG_COHORT_CREATION: 'feature-flag-cohort-creation', // owner: #team-feature-flags
```

Comment:
btw this `feature-flag-cohort-creation` feature is really close to working so I want to keep this flag around since it's likely we could ship this soon.

Reply:
Can you move it to the WIP section then? This way I wont bother you the next time I come around and try cleaning this up :)